src/pages/_app.tsx

```diff
@@ -132,6 +132,19 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
       {getLayout(<Component {...pageProps} />)}
 
       <Toaster />
+
+      <div
+        id="idos_container"
+        style={{
+          position: 'absolute',
+          top: '50%',
+          left: '50%',
+          width: '150px',
+          height: '30px',
+          margin: 0,
+          padding: 0,
+        }}
+      />
```

Comment:
We should update this CSS to be:

```
style={{
          position: 'absolute',
          top: 0,
          left: 0,
          width: 0,
          height: 0,
          margin: 0,
          padding: 0,
          opacity: 0,
        }} 
```

Currently the DIV sits right in the middle of the page and actually blocks the UI (I added a red background to it so you can see):

<img width="1697" alt="Screen Shot 2023-11-03 at 2 21 22 PM" src="https://github.com/near/near-discovery/assets/1475067/7da9430c-2ba4-4985-a9dd-6540b11264c6">

This `<div>` shouldn't be visible or have any width/height. It just need to exist so the SDK can insert an `<iframe>`.